### ISSUE TITLE
Help Menu Readme Error

### ISSUE THREAD

**Issue by Rcrash88** [projectM-visualizer/frontend-sdl2#64]
Please correct the "Use F1 for help menu" line in Readme.md to "Press ESC To Access the Help Toolbar."

**Comment by kblaschke** [comment]
The whole UI (menu, hotkeys etc.) is still very much a work in progress, and stuff changes all the time. So the current documentation may not reflect the state of the code, as it would cost lots of time to always update the docs for each code change.

That said, we'll be updating all documentation and user manuals when the new UI is done and working and the hotkeys won't change anymore until the final proper release.

**Comment by kblaschke** [comment]
Yes, the whole text menu was broken in the last few versions, and then removed completely. We're working on a new UI with way more functionality right now, was the a final release planned early to mid 2024. Then the documentation and everything else will be updated to reflect the state of the application. Hopefully, distribution packagers will also pick up the new app and libprojectM then, so the situation improves for everyone.
